apps/web/pages/utils/last.mdx

```diff
@@ -0,0 +1,54 @@
+# last
+
+Get the last item of an array.
+
+## Installation
+
+import { Tab, Tabs } from 'nextra-theme-docs';
+
+{/* prettier-ignore */}
+<Tabs items={['npm', 'yarn', 'pnpm']}>
+    <Tab>
+        ```bash copy
+        npm install @madeinhaus/utils
+        ```
+    </Tab>
+    <Tab>
+        ```bash copy
+        yarn add @madeinhaus/utils
+        ```
+    </Tab>
+    <Tab>
+        ```bash copy
+        pnpm add @madeinhaus/utils
+        ```
+    </Tab>
+</Tabs>
+
+## Import
+
+```tsx copy
+import { last } from '@madeinhaus/utils';
+```
+
+## Parameters
+
+- _(array)_: The array to get the last item from.
+
+## Returns
+
+- _(number)_: The last item of the array, or `undefined` if the array is empty.
```

Comment:
```suggestion
- The last item of the array, or `undefined` if the array is empty.
```
The return type is `T`, i think it's probably best to remove the type altogether.